src/main/resources/templates/admin/organization.html

```diff
@@ -75,7 +75,7 @@ <h4>Create Planting Site</h4>
     <br/>
     <label for="zipfile">Zipfile</label>
     <input type="file" name="zipfile" id="zipfile" required/>
-    Must contain three shapefiles and their associated supplementary datafiles.
+    Must contain four shapefiles and their associated supplementary datafiles.
```

Comment:
Should this actually be three?

Reply:
Good catch, thanks. You can tell my initial stab at this was to load monitoring plots from the shapefiles!